python/src/Graph.i

```diff
@@ -3,58 +3,32 @@
 %{
 #include "openturns/Graph.hxx"
 
-namespace OT { 
+namespace OT {
 
   template <>
   struct traitsPythonType<OT::Graph>
   {
-    typedef _PySequence_ Type;
+    typedef _PyObject_ Type;
   };
 
   template <>
   inline
   OT::Graph
-  convert<_PySequence_,OT::Graph>(PyObject * pyObj)
+  convert<_PyObject_, OT::Graph>(PyObject * pyObj)
```

Comment:
Is this block still needed?  It looks like it could be dropped.

Reply:
Yes